packages/tupaia-web/src/constants/dashboardItems.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+export const DashboardItemVizTypes = {
+  Matrix: 'matrix',
+  Chart: 'chart',
+  View: 'view',
+};
```

Comment:
It's a shame about the enum issue. Potentially we could think about introducing some global constants to either the types package, or maybe a separate common package. But this is fine for now :smile_cat: